src/ApplicationRunner.php

```diff
@@ -22,75 +22,27 @@
  */
 abstract class ApplicationRunner implements RunnerInterface
 {
-    protected ?ConfigInterface $config = null;
-    protected ?ContainerInterface $container = null;
-    protected ?string $bootstrapGroup = null;
-    protected ?string $eventsGroup = null;
+    private ?ConfigInterface $config = null;
+    private ?ContainerInterface $container = null;
 
     /**
      * @param string $rootPath The absolute path to the project root.
      * @param bool $debug Whether the debug mode is enabled.
-     * @param string $paramsConfigGroup The config parameters group name.
-     * @param string $containerConfigGroup The container configuration group name.
+     * @param string|null $configGroupPostfix A configuration groups postfix.
      * @param string|null $environment The environment name.
      */
     public function __construct(
         protected string $rootPath,
         protected bool $debug,
-        protected string $paramsConfigGroup,
-        protected string $containerConfigGroup,
-        protected ?string $environment
+        protected bool $useBootstrap,
```

Comment:
Do we need an ability not to use it?

Reply:
Hm, probably not. Check on exists config group is enough. I will remove this option.

Reply:
done